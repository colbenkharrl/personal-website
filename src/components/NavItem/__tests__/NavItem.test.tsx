import { render, screen } from '@testing-library/react';
import { NavItem } from '../';

it('NavItem matches snapshot (regression test)', () => {
  render(<NavItem href="https://www.example.com">I am a link.</NavItem>);
  expect(screen.getByTestId('NavItem')).toMatchSnapshot(
    'Full component snapshot.',
  );
});
