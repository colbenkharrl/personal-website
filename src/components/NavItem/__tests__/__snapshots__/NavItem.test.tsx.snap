// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavItem matches snapshot (regression test): Full component snapshot. 1`] = `
<a
  class="relative block px-3 py-2 transition hover:text-orange-500 dark:hover:text-orange-400"
  data-testid="NavItem"
  href="https://www.example.com"
>
  I am a link.
</a>
`;
